modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/wal/WalCompactionAfterRestartTest.java

```diff
@@ -40,11 +40,13 @@
 import org.apache.ignite.lang.IgniteBiTuple;
 import org.apache.ignite.testframework.GridTestUtils;
 import org.apache.ignite.testframework.junits.common.GridCommonAbstractTest;
+import org.junit.Ignore;
 import org.junit.Test;
 
 import static org.apache.ignite.events.EventType.EVT_WAL_SEGMENT_COMPACTED;
 
 /** */
+@Ignore("FAILED!")
```

Comment:
Please file a separate ticket to deal with all failed tests, link it here

Reply:
@alamar, I do remember you just commented tests at suites at your similar cleanup work.
Let's define some markers like @Ignore("TBD") to make this fix possible in 2020.

Reply:
It was a compromise that was explicitly discussed, which included creation of separate tickets, so I think we should have tickets here as well.

Reply:
Same problem here, issue creation is an optional requirement.

Reply:
Added jira url.